src/index.js

```diff
@@ -0,0 +1,132 @@
+const state = {
+  temp: 68
+};
+
+let isFahrenheit = true;
+
+const vibe = {
+  sheet: 2
+}
+
+let isReset = false;
+
+const changeTemp = (direction) => {
+  if (direction === 'up') {
+    state.temp += 1;
+  } else if (direction === 'down') {
+    state.temp -= 1;
+  } else if (direction === null) {
+    pass
+  }
+
+  const temperature = document.querySelector("#temperature");
```

Comment:
Consider moving the part of the code here that applies the new values to the UI into a separate helper function. This would allow that part to be called from other code that needs to update the UI, but doesn't need to modify the temperate (e.g., when the units change).